pkg/packet/bgp/bgp.go

```diff
@@ -10029,8 +10032,8 @@ func ParseExtendedCommunity(subtype ExtendedCommunityAttrSubType, com string) (E
 	}
 }
 
-func ParseRouteTarget(rt string) (ExtendedCommunityInterface, error) {
```

Comment:
This function is publicly exported. You can't change it without the major version updated.

Reply:
Fixed.
Thanks